packages/data-stores/src/site/reducer.ts

```diff
@@ -101,12 +101,15 @@ export const sitesDomains: Reducer< { [ key: number ]: Domain[] }, Action > = (
 	return state;
 };
 
-export const launchStatus: Reducer< { [ key: number ]: boolean }, Action > = (
-	state = {},
-	action
-) => {
-	if ( action.type === 'LAUNCHED_SITE' ) {
-		return { ...state, [ action.siteId ]: true };
+export const launchStatus: Reducer<
+	{ [ key: number ]: { isLaunched: boolean; isSiteLaunching: boolean } },
```

Comment:
We should probably name variables similarly — e.g `isSiteLaunched` and `isSiteLaunching`, or `isLaunched` and `isLaunching`

Reply:
done in https://github.com/Automattic/wp-calypso/pull/47601/commits/a1d1ff86292dd085e7727e408c58c58c0ac8a684